src/index.ts

```diff
@@ -102,6 +102,7 @@ function init(modules: { typescript: typeof ts_module }) {
             length: problem.getEndPosition().getPosition() - problem.getStartPosition().getPosition(),
             messageText: message,
             category: category,
+            source: 'tslint',
```

Comment:
Since all the diagnostics will all come from the same version the base `tslint` is sufficient.